test/controllers/transport/WhichRailwayTradeControllerSpec.scala

```diff
@@ -18,16 +18,21 @@ package controllers.transport
 
 import base.SpecBase
 import forms.transport.WhichRailwayTradeFormProvider
-import models.{NormalMode, UserAnswers, WhichRailwayTrade}
+import generators.Generators
+import models.{NormalMode, ThirdIndustryOptions, UserAnswers, WhichRailwayTrade}
 import navigation.{FakeNavigator, Navigator}
+import org.scalatest.concurrent.{IntegrationPatience, ScalaFutures}
+import org.scalatest.prop.PropertyChecks
+import pages.ClaimAmount
 import pages.transport.WhichRailwayTradePage
 import play.api.inject.bind
 import play.api.mvc.Call
 import play.api.test.FakeRequest
 import play.api.test.Helpers._
+import repositories.SessionRepository
 import views.html.transport.WhichRailwayTradeView
 
-class WhichRailwayTradeControllerSpec extends SpecBase {
+class WhichRailwayTradeControllerSpec extends SpecBase with ScalaFutures with IntegrationPatience with PropertyChecks with Generators {
```

Comment:
"with IntegrationPatience with PropertyChecks with Generators" 
Because we are not using Generators in this spec I think we can remove a couple of these as they are not used.

Reply:
Done